game/game_module.rb

```diff
@@ -1,31 +1,21 @@
 module GameModule
-  def read_input(message)
-    print "#{message}: "
-    gets.chomp
-  end
-
-  def add_game
-    multiplayer = read_input('Is game multiplayer? [true, false]')
-    last_played_at = read_input('Game Last played at [yyyy-mm-dd]')
-    publish_date = read_input('Game publish at [yyyy-mm-dd]')
-
-    author_first_name = read_input('Author first name')
-    author_last_name = read_input('Author last name')
-
+  def add_game(multiplayer, last_played_at, publish_date, author_first_name, author_last_name)
     author = Author.new(author_first_name, author_last_name)
     new_game = Game.new(multiplayer, last_played_at, publish_date)
     new_game.author = author
 
     add_author(author)
-    save_game(new_game)
-    puts 'Game has been created successfully'
-  end
-
-  def save_game(new_game)
+    # save_game(new_game)
     @games.push({ 'multiplayer' => new_game.multiplayer, 'last_played_at' => new_game.last_played_at,
                   'publish_date' => new_game.publish_date })
+    'Game has been created successfully'
   end
 
+  # def save_game(new_game)
+  #   @games.push({ 'multiplayer' => new_game.multiplayer, 'last_played_at' => new_game.last_played_at,
+  #                 'publish_date' => new_game.publish_date })
+  # end
```

Comment:
Hi @gak92 

This is optional, you can remove this comment to make the code clean since it is no longer needed now

Reply:
@Owusu-Desmond,
Sure, I will do that.